docs/example-scenario/data/big-data-with-iot.md

```diff
@@ -0,0 +1,90 @@
+            
+## Potential use cases
+
+Consider this solution for the following use cases:
+
+* Construction, mining, or equipment manufacturing scenarios
+* Large-scale collection of device data for storage and analysis
+* Ingestion and analysis of large datasets 
+
+## Architecture
+
+![Architecture for IoT and data analytics in the construction industry][architecture]
+
+The data flows through the solution as follows:
+
+1. Construction equipment collects sensor data and sends the construction results data at regular intervals to load balanced web services hosted on a cluster of Azure virtual machines.
+2. The custom web services ingest the construction results data and store it in an Apache Cassandra cluster also running on Azure virtual machines.
+3. Another dataset is gathered by IoT sensors on various construction equipment and sent to IoT Hub.
+4. Raw data collected is sent directly from IoT Hub to Azure blob storage and is immediately available for viewing and analysis.
+5. Data collected via IoT Hub is processed in near real time by an Azure Stream Analytics job and stored in an Azure SQL database.
+6. The Smart Construction Cloud web application is available to analysts and end users to view and analyze sensor data and imagery. 
+7. Batch jobs are initiated on demand by users of the web application. The batch job runs in Apache Spark on HDInsight and analyzes new data stored in the Cassandra cluster. 
+
+### Components
+
+* [IoT Hub](/azure/iot-hub) acts as a central message hub for secure bi-directional communication with per-device identity between the cloud platform and the construction equipment and other site elements. IoT Hub can rapidly collect data for each device for ingestion into the data analytics pipeline. 
+* [Azure Stream Analytics](/azure/stream-analytics) is an event-processing engine that can analyze high volumes of data streaming from devices and other data sources. It also supports extracting information from data streams to identify patterns and relationships. In this scenario, Stream Analytics ingests and analyzes data from IoT devices and stores the results in Azure SQL Database. 
+* [Azure SQL Database](/azure/sql-database) contains the results of analyzed data from IoT devices and meters, which can be viewed by analysts and users via an Azure-based Web application. 
+* [Azure Blob storage](/azure/storage/blobs) stores image data gathered from the IoT hub devices. The image data can be viewed via the web application.
+* [Traffic Manager](/azure/traffic-manager) controls the distribution of user traffic for service endpoints in different Azure regions.
+* [Load Balancer](/azure/load-balancer) lets you scale your applications and create high availability for your services. The load balancer distributes data submissions from construction equipment devices.
```

Comment:
> lets you scale your applications

Could be confusing whether "application" means the whole solution or the app running on the VMs.

Maybe say "scale out VMs" or "distribute load across VMs"? (or: Scale out the web application?)

Reply:
Fixed.